tests/__mocks__/fileMock.ts

```diff
@@ -0,0 +1,7 @@
+// This file, along the `moduleNameMapper` field found in the jest config,
+// is meant to fix the jest issue regarding assets imports during React component testing.
+//
+// For more information, please refer to this thread:
+// https://stackoverflow.com/questions/46898638/importing-images-breaks-jest-test
+
+export default "";
```

Comment:
I... alright

Reply:
Yeah.. This makes no sense but that's the only thing that fixed the issue (hence the comment because this is really confusing).